pkg/minikube/update/update_test.go

```diff
@@ -1,183 +0,0 @@
-/*
```

Comment:
No sense to keep this. In fact it was even disabled at the time of fork. If we go to write tests we most likely start from scratch anyways.